auth-server/src/main/java/org/hyperskill/community/authserver/config/MongoConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package org.hyperskill.community.authserver.config;
+
+import com.mongodb.client.MongoClients;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import java.util.Objects;
+
+@Configuration
+@Slf4j
+public class MongoConfiguration {
+
+    @Bean
+    public MongoTemplate mongoTemplate(Environment env) {
+        var host = env.getProperty("MONGO_HOST");
```

Comment:
I wonder can we set this in application.properties file with default values? Like `spring.data.mongodb.host=${MONGO_HOST:localhost}`Not a critical thing, though.

Reply:
I would not do that - see no benefit. It is one more property, that only indirects the use